### ISSUE TITLE
The sign in dielectric tensors

### ISSUE THREAD

**Issue by FPrith** [kumagai-group/vise#13]
According to vise commands and “Commands for creating inputs” in Wiki, I tried evaluating the dielectric tensors in some insulating oxides.
Some approximations for evaluating dielectric tensors were tried, including the random phase approximation (RPA).

The values in cubic systems were comparable to those reported previously.
Meanwhile, for example, in monoclinic systems (ZrO2, HfO2, Bi2O3), OUTCAR files showed the positive sign of ε(13) components although their negative signs were reported in a Prof. Kumagai’s article and other articles.
https://journals.aps.org/prb/abstract/10.1103/PhysRevB.89.195205
This difference in the sign of components seems not to be related to XC functionals.

Below is an example that was obtained in my calculation for monoclinic HfO2.
･ VASP version: 5.4.4
･ XC functional: PBEsol
･ Ecut = 520 eV
･ k-points mesh: 10 * 10 * 10
･ Origin in BZ sampling: (0.5, 0.5, 0.5), gamma-centered mesh

 MACROSCOPIC STATIC DIELECTRIC TENSOR (including local field effects in DFT)
 ------------------------------------------------------
           4.979547     0.000000     0.140013
           0.000000     5.001772     0.000000
           0.140012     0.000000     4.693212
 ------------------------------------------------------

I would appreciate it if someone would tell me why this difference appears and what calculation processes/conditions are needed to obtain correct dielectric tensors.

**Comment by yuuukuma** [comment]
Thanks. 
The difference of the off-diagonal components is due to the difference of the unit cells.
In our previous study, we used different unitcell, which will change the sign of the off-diagonal components.

**Comment by FPrith** [comment]
Thank you for your reply.
What is the crystallographic difference between the unit cells that you mentioned in the case of monoclinic HfO2?

I did additional calculations for ZrO2 cells with the P21/c space group.
Some kinds of primitive cells with different lattice constants and real-space vectors taken from ICSD and Materials Project were used, but no negative sign of ε(13) was obtained.

**Comment by yuuukuma** [comment]
Did you calculate the structure where the angle of beta is changed to (180 - beta)?
Then, I think you obtain the sign inverted results.

**Comment by FPrith** [comment]
Your indication is that, in monoclinic cells, an obtuse angle of beta is needed, isn't it?
My calculations of dielectric tensors were done on obtuse-beta cells.

**Comment by yuuukuma** [comment]
Year, in general, the sign of the off-diagonal component depends on the angle of beta (acute or obtuse) and also on the atomic arrangement.

Anyway, here I attach the POSCAR file that I calculated in the previous paper https://journals.aps.org/prb/abstract/10.1103/PhysRevB.89.195205. 
(I received this POSCAR file from my colleague,  and so I don't know why a, b are set along the negative x, y directions.)

```
ZrO2/O_m2_2_Zr_p4_1/Hf_p4_1_O_m2_2
   1.00000000000000
    -5.0990038581716863    0.0000000000000000   -0.0090009036420896
     0.0000000000000000   -5.1494227868959257    0.0000000000000000
     0.8785689769388899    0.0000000000000000    5.1917281804172051
   Hf   O_
     4     8
Direct
  0.7239371709721496  0.4579216888387023  0.7928012064989503
  0.2760628290278504  0.9579216888387023  0.7071987935010497
  0.2760628290278504  0.5420783111612977  0.2071987935010497
  0.7239371709721496  0.0420783111612977  0.2928012064989503
  0.0687190200621117  0.6686403652373016  0.8462062643491208
  0.9312809799378883  0.1686403652373016  0.6537937356508792
  0.9312809799378883  0.3313596347626984  0.1537937356508792
  0.0687190200621117  0.8313596347626984  0.3462062643491208
  0.4494148160768745  0.2425636581523207  0.9798152625809706
  0.5505851839231255  0.7425636581523207  0.5201847374190294
  0.5505851839231255  0.7574363418476793  0.0201847374190294
  0.4494148160768745  0.2574363418476793  0.4798152625809706
```

**Comment by FPrith** [comment]
Thank you for sharing a POSCAR example.
Do the signs of off-diagonal components in monoclinic cells and triclinic ones affect sperically avaraged ε(ele) to be used for the calculations using sc-dd or nsc-dd hybrid functionals?
My concern is on whether the electronic states evaluated by dd hybrid functionals differ depending on POSCAR files or not.

**Comment by yuuukuma** [comment]
> Do the signs of off-diagonal components in monoclinic cells and triclinic ones affect sperically avaraged ε(ele) to be used for the calculations using sc-dd or nsc-dd hybrid functionals?

No, because the spherically averaged value is calculated by (e11 + e22 + e33)/3.

**Comment by FPrith** [comment]
Thank you for reply.
I will do additional calculations in monoclinic (and triclinic) cells and attempt to understand the relation between POSCAR files created via some tools and values of off-diagonal components in OUTCAR files.
